css/bs-example.css

```diff
@@ -0,0 +1,138 @@
+/*
+ * Examples
+  *
+   * Isolated sections of example content for each component or feature. Usually
+    * followed by a code snippet.
+     */
+
+     .bs-example {
+         position: relative;
+           padding: 15px 15px 15px;
+             margin: 0 -15px 15px;
+               background-color: #fafafa;
+                 box-shadow: inset 0 3px 6px rgba(0,0,0,.05);
+                   border-color: #e5e5e5 #eee #eee;
+                     border-style: solid;
+                       border-width: 1px 0;
```

Comment:
What happened here?  Can this indentation be cleaned up to be more readable?

Reply:
um...idk

Reply:
what should I fix again?